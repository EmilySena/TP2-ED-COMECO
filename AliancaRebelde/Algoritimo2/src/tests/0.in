50
OrBVohrvpC 715 431773
eirEivyvAa 715 165540
KjJHlUcVnz 715 352595
OEUiianXXZ 715 318947
JXGfosDsye 66804 113160
MFxpjSanWK 715 487824
zMzmvUGuGh 66804 17828
bDSGcZIaKq 715 969366
CNlLGWWKQe 66804 445183
kcGCukKfWq 66804 574020
HuRuUNaezV 66804 768210
xpRqoYGcGJ 715 678948
aLxfhRCqIp 715 519942
ibYDhGlHzM 715 830427
LpElaojxaF 715 864609
cWmoVjYpDf 66804 567283
OfQcAwuwXi 715 196891
SSMqcEYktN 66804 727141
LpHERtAdzy 715 73227
OXwNeawLFx 66804 25641
ueckQiHEJV 66804 618270
NVDOWtumBS 715 536959
UIWKyflVhA 66804 893225
uuCqfpqyWR 66804 169839
JEJEQzTIUX 715 611235
gVrIvwzugU 66804 793643
vKvGGIqIvN 715 267026
KZXcBGeDGY 715 434566
JYYdojqsrw 66804 926743
rDvdAAKxpB 66804 777022
qslREMheCu 715 798784
yMqlPlGaDe 66804 667700
tTXmWAgmYH 715 799979
HtZmRgyItd 66804 707371
LjCcTzhwjx 66804 23219
KnbSzMzPwW 715 697292
OYWFlPwKOS 715 299086
OSpxYKcOPo 715 268765
rBxEuvHACu 715 238474
KzoXdmDoDd 66804 108753
wwtbyStoDA 715 823399
LKbkqxicwT 66804 528992
OLfbxiTyeZ 66804 903846
CeUkDQvPUb 66804 925552
KOkLTrvdMu 715 840337
dybfwTOIkH 66804 190358
cVtaKuvjrX 66804 532902
FLIwCXNjMS 66804 103848
FwUkpQjBnr 715 643962
LnVRaCEeUz 66804 198720
