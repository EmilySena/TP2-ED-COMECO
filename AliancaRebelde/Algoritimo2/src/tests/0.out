bDSGcZIaKq 715 969366
LpElaojxaF 715 864609
KOkLTrvdMu 715 840337
ibYDhGlHzM 715 830427
wwtbyStoDA 715 823399
tTXmWAgmYH 715 799979
qslREMheCu 715 798784
KnbSzMzPwW 715 697292
xpRqoYGcGJ 715 678948
FwUkpQjBnr 715 643962
JEJEQzTIUX 715 611235
NVDOWtumBS 715 536959
aLxfhRCqIp 715 519942
MFxpjSanWK 715 487824
KZXcBGeDGY 715 434566
OrBVohrvpC 715 431773
KjJHlUcVnz 715 352595
OEUiianXXZ 715 318947
OYWFlPwKOS 715 299086
OSpxYKcOPo 715 268765
vKvGGIqIvN 715 267026
rBxEuvHACu 715 238474
OfQcAwuwXi 715 196891
eirEivyvAa 715 165540
LpHERtAdzy 715 73227
JYYdojqsrw 66804 926743
CeUkDQvPUb 66804 925552
OLfbxiTyeZ 66804 903846
UIWKyflVhA 66804 893225
gVrIvwzugU 66804 793643
rDvdAAKxpB 66804 777022
HuRuUNaezV 66804 768210
SSMqcEYktN 66804 727141
HtZmRgyItd 66804 707371
yMqlPlGaDe 66804 667700
ueckQiHEJV 66804 618270
kcGCukKfWq 66804 574020
cWmoVjYpDf 66804 567283
cVtaKuvjrX 66804 532902
LKbkqxicwT 66804 528992
CNlLGWWKQe 66804 445183
LnVRaCEeUz 66804 198720
dybfwTOIkH 66804 190358
uuCqfpqyWR 66804 169839
JXGfosDsye 66804 113160
KzoXdmDoDd 66804 108753
FLIwCXNjMS 66804 103848
OXwNeawLFx 66804 25641
LjCcTzhwjx 66804 23219
zMzmvUGuGh 66804 17828
